exec/cnex/bytecode.h

```diff
@@ -15,6 +15,38 @@ typedef struct tagTFunction {
     unsigned int entry;
 } Function;
 
+typedef struct tagTExportConstant {
+    unsigned int name;
+    unsigned int vtype;
+    unsigned char *value;
+} Constant;
+
+typedef struct tagTVariable {
+    unsigned int name;
+    unsigned int type;
+    unsigned int index;
+} Variable;
+
+//type exportfunction struct {
```

Comment:
What's this? It does not seem to belong here.

Reply:
Removed.